### DISCUSSION TITLE
530 login incorrect

### DISCUSSION THREAD

**Discussion by charlinelanc** [iterate-ch/cyberduck#14638] [Q&A]
![Capture d’écran 2023-05-07 à 20 27 16](https://user-images.githubusercontent.com/132932100/236880234-c380fa32-275c-402f-8321-f2f09d21b6df.png)
Hello,
I can no longer access my account. name and password are correct. gazier is unreachable. What should I do ?
Tkxss
Ch.

**Comment by dkocher** [comment]
If you have trouble connecting to a server or your login credentials are not valid, try to resolve the issue with the assistance of your hosting service provider.
